toolchain/lowering/lowering_function_context.h

```diff
@@ -40,6 +40,11 @@ class LoweringFunctionContext {
 
   // Returns a local (versus global) value for the given node.
   auto GetLocal(SemanticsNodeId node_id) -> llvm::Value* {
+    // All builtins are types, with the same empty lowered value.
+    if (node_id.index < SemanticsBuiltinKind::ValidCount) {
+      return GetTypeValue();
+    }
```

Comment:
GetTypeValue is documented as returning the value for `type`. Should this only handle TypeType, or should the function documentation be fixed?

Consider a name like GetValueForBuiltinType if this needs to handle more than TypeType

Reply:
Per discussion, this is the value used for *any* type.